pandas_gbq/constants.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 pandas-gbq Authors All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+BYTES_IN_KIB = 1024
+BYTES_IN_MIB = 1024 * BYTES_IN_KIB
+BYTES_IN_GIB = 1024 * BYTES_IN_MIB
+BYTES_TO_RECOMMEND_BIGFRAMES = BYTES_IN_GIB
```

Comment:
Are these names governed by pandas? If not, can we use KB, MB, GB instead? Totally a nit and not a big deal if this is a convention for pandas.

Reply:
Not by pandas, but BigQuery. See: https://cloud.google.com/bigquery/pricing#storage and https://cloud.google.com/bigquery/pricing#data

> The size of your stored data and the size of the data processed by your queries is calculated in [gibibytes (GiB)](https://en.wikipedia.org/wiki/Gibibyte), where 1 GiB is 230 bytes (1,024 MiB). Similarly, 1 tebibyte (TiB) is 240 bytes (1,024 GiB).